### PR TITLE
Attempt to create Keep-playing option

### PR DESCRIPTION
**Pull request by ministro**
#### What is it?
- [ ] Bugfix (user facing)
- [x ] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

- record videos
- create clones
- take over the world

#### Before/After Screenshots/Screen Record

- Before:
- After:

#### Fixes the following issue(s)

- Fixes #

#### Relies on the following changes

- 

#### APK testing 

The APK can be found by going to the "Checks" tab below the title. On the left pane, click on "CI", scroll down to "artifacts" and click "app" to download the zip file which contains the debug APK of this PR.

#### Due diligence
- [ ] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by ministro** [comment]
I have no idea how to build this nor if this would be enough, but I'd love to see this simple feature in a future release.

File: app/src/main/java/org/schabi/newpipe/player/helper/PlayerHelper.java [hunk 1]
```diff
@@ -249,6 +249,11 @@ public static boolean isResumeAfterAudioFocusGain(@NonNull final Context context
         return getPreferences(context)
                 .getBoolean(context.getString(R.string.resume_on_audio_focus_gain_key), false);
     }
+        
+    public static boolean isPauseAfterAudioFocusLoss(@NonNull final Context context) {
+        return getPreferences(context)
+                .getBoolean(context.getString(R.string.pause_on_audio_focus_loss_key), true);
```
Comment (ministro) [inline app/src/main/java/org/schabi/newpipe/player/helper/PlayerHelper.java:255]: I assume we would want to keep the default behaviour the same.

**Comment by opusforlife2** [comment]
Please fill out the template properly.

**Comment by opusforlife2** [comment]
Closing: https://github.com/TeamNewPipe/NewPipe/issues/8543#issuecomment-1166365385

**Pull request closed without merging**